autogluon/task/object_detection/object_detection.py

```diff
@@ -295,7 +295,8 @@ def fit(dataset='voc',
         args = sample_config(train_object_detection.args, results['best_config'])
         logger.info('The best config: {}'.format(results['best_config']))
 
-        ctx = [mx.gpu(i) for i in range(get_gpu_count())]
+        ngpus = get_gpu_count()
+        ctx = [mx.gpu(i) for i in range(ngpus)] if ngpus > 0 else [mx.cpu()]
```

Comment:
@mak-454 After modularization, this file has been moved to `vision/src/autogluon/vision/object_detection/object_detection.py`, can you update the code? I can help review the change again!

Reply:
@zhreshold sure will do.

Reply:
@zhreshold please review.